src/parameters.toml

```diff
@@ -763,3 +763,221 @@ alias = "avogad"
+value = 0.25
+type = "float"
+explanation = "Critical gradient Richardson number. It is an upper limit to the gradient Richardson number . See Li (2019) [https://doi.org/10.1016/j.atmosres.2018.09.015], Section 6.2 for details"
+
+[mixing_length_smin_ub]
+alias = "smin_ub"
+value = 0.1
+type = "float"
+
+[mixing_length_smin_rm]
+alias = "smin_rm"
+value = 1.5
+type = "float"
+
+[mixing_length_l_max]
+alias = "l_max"
+value = 1.0e6
+type = "float"
+
+[entrainment_factor]
+alias = "entrainment_factor"
+value = 0.13
+type = "float"
+
+[detrainment_factor]
+alias = "detrainment_factor"
+value = 0.51
+type = "float"
+
+[entrainment_massflux_div_factor]
+alias = "entrainment_massflux_div_factor"
+value = 0.0
+type = "float"
+
+[turbulent_entrainment_factor]
+alias = "turbulent_entrainment_factor"
+value = 0.075
+type = "float"
+
+[entrainment_smin_tke_coeff]
+alias = "entrainment_smin_tke_coeff"
+value = 0.3
+type = "float"
+
+[updraft_mixing_frac]
+alias = "updraft_mixing_frac"
+value = 0.25
+type = "float"
+
+[entrainment_area_limiter_scale]
+alias = "area_limiter_scale"
+value = 10.0
+type = "float"
+
+[entrainment_area_limiter_power]
+alias = "area_limiter_power"
+value = 4.0
+type = "float"
+
+[entrainment_scale]
+alias = "entrainment_scale"
+value = 0.0004
+type = "float"
+
+[entrainment_sorting_power]
+alias = "sorting_power"
+value = 2.0
+type = "float"
+
+[minimum_updraft_velocity]
+alias = "min_upd_velocity"
+value = 0.001
+type = "float"
+
+[minimum_updraft_top]
+alias = "min_updraft_top"
+value = 500.0
+type = "float"
+
+[pressure_normalmode_buoy_coeff1]
+alias = "pressure_normalmode_buoy_coeff1"
+value = 0.12
+type = "float"
+
+[pressure_normalmode_buoy_coeff2]
+alias = "pressure_normalmode_buoy_coeff2"
+value = 0.0
+type = "float"
+
+[pressure_normalmode_adv_coeff]
+alias = "pressure_normalmode_adv_coeff"
+value = 0.1
+type = "float"
+
+[pressure_normalmode_drag_coeff]
+alias = "pressure_normalmode_drag_coeff"
+value = 10.0
+type = "float"
+
+[EDMF_thermodynamics_moisture_model]
+alias = "moisture_model"
+value = "equilibrium"
+type = "string"
+
+[EDMF_thermodynamics_covariance_model]
+alias = "thermo_covariance_model"
+value = "diagnostic"
+type = "string"
+
+[EDMF_thermodynamics_diagnostic_covar_limiter]
+alias = "diagnostic_covar_limiter"
+value = 0.001
+type = "float"
+
+[EDMF_thermodynamics_sgs]
+alias = "sgs"
+value = "mean"
+type = "string"
+
+[EDMF_thermodynamics_quadrature_order]
+alias = "quadrature_order"
+value = 3
+type = "float"
+
+[EDMF_thermodynamics_quadrature_type]
+alias = "quadrature_type"
+value = "log-normal"
+type = "string"
+
+[updraft_number]
+alias = "updraft_number"
+value = 1
+type = "float"
+
+[pressure_closure_buoy]
+alias = "pressure_closure_buoy"
+value = "normalmode"
+type = "string"
+
+[pressure_closure_drag]
+alias = "pressure_closure_drag"
+value = "normalmode"
+type = "string"
+
+[microphysics_model_precipitation_fraction]
+alias = "precip_fraction_model"
+value = "prescribed"
+type = "string"
```

Comment:
explanation = "Precipitation fraction model choice. Options are prescribed (default) and cloud_cover. Valid for simulations running with the 1-moment microphysics scheme, edmf turbulence convection model. Not documented in the literature (yet)."